layers/meta-balena-beaglebone/recipes-core/images/balena-image.inc

```diff
@@ -2,6 +2,8 @@ IMAGE_FSTYPES:append:beaglebone = " balenaos-img balenaos-img.bmap"
 IMAGE_FSTYPES:append:beaglebone-pocket = " balenaos-img"
 IMAGE_FSTYPES:append:beagleboard-xm = " balenaos-img"
 
+BALENA_BOOT_SIZE:beagleplay = "327680"
```

Comment:
Again, ~100MB should be more than sufficient for the boot partition

Reply:
See the comment above.